lib/Twig/Node/Empty.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ * (c) Armin Ronacher
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Dummy node that represents an empty node, not producing any output.
+ *
+ * @author Denis Brumann <denis.brumann@sensiolabs.de>
+ */
+class Twig_Node_Empty extends Twig_Node
```

Comment:
This node must implement `Twig_NodeCaptureInterface`, otherwise `{% use %}` will be forbidden at top-level in templates extending another template.

And the class should be final to forbid turning an empty node into a child class not being empty